libs/tracing/ppx/ppx_tracing.ml

```diff
@@ -0,0 +1,115 @@
+(* Emma Jin
+ *
+ * Copyright (C) 2023 Semgrep, Inc
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation, with the
+ * special exception on linking described in file license.txt.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
+ * license.txt for more details.
+ *)
+open Ppxlib
+open Ast_helper
+
+(*****************************************************************************)
```

Comment:
maybe add a Prelude section explaining what this module is, and maybe
that this is mostly a copy of ppx_profiling but adapted to use ocaml-trace